docs/tutorials/multinode/circuit_q24.sub

```diff
@@ -0,0 +1,11 @@
+universe                = docker
```

Comment:
See comment in `multinode/README.md` - this and `circuit_q30.sub` are based on the old text format, and should be removed or replaced with Cirq-based `.sub` files.